x/merkledb/db.go

```diff
@@ -422,6 +422,37 @@ func (db *merkleDB) Close() error {
 	return db.baseDB.Put(cleanShutdownKey, hadCleanShutdown)
 }
 
+func (db *merkleDB) PrefetchKeys(keys [][]byte) error {
```

Comment:
I'm thinking we should call this something like `WarmPath`? What we want from this is to load intermediate nodes into memory so we can more quickly generate the root.